src/form-explainer/closing-disclosure.html

```diff
@@ -23,7 +23,7 @@ <h1 class="page-title">Closing Disclosure Explainer</h1>
       </div><!-- /.col-8.page-intro -->
 
       <div class="col-4 illustration">
-        <img src="../static/img/ill-form-explainer.png" alt="">
+        <img src="../static/img/ill-form-explainer-closing-disc.png" alt="Image of the Closing Disclosure mortgage form">
```

Comment:
Since this is an illustration and not an image of the actual form, I'd argue it's decorative and the alt text isn't helpful information for those with images off or using screen readers..

from W3C standards:

>Purely decorative images are visual enhancements, decorations or embellishments that provide no function or information beyond aesthetics to users who can view the images... Mark up purely decorative images so they can be ignored by assistive technology by using an empty alt attribute (alt="").